.sage/proto.go

```diff
@@ -67,7 +60,7 @@ func (Proto) BufGenerateGoogleapis(ctx context.Context) error {
 	cmd := sgbuf.Command(
 		ctx,
 		"generate",
-		"buf.build/googleapis/googleapis:"+googleapisRef,
+		"https://github.com/googleapis/googleapis.git#branch=master",
```

Comment:
Couldn't this lead to an unstable CI, as there could be changes (e.g. added fields) to the files that we generate from here 🤔

Reply:
Yes, it will. See my comment above.